metrics/lib/send_results.sh

```diff
@@ -36,7 +36,7 @@ COR_DEFAULT_PATH="/usr/bin"
 COR_DEFAULT_NAME="cc-oci-runtime"
 RUNC_DEFAULT_PATH="/usr/bin"
 RUNC_DEFAULT_NAME="docker-runc"
-CC_RUNTIME_DEFAULT_PATH="/usr/local/bin"
+CC_RUNTIME_DEFAULT_PATH="/usr/bin"
```

Comment:
I agree we could be smarter in our search. There is a limit to how smart I think we should be though.
Given that we have two defaults in our build files afaik, then maybe we should do a dual-search here.
An, sure, printing out some diagnostics could be useful (but, otoh, we call this code a lot when running the metrics to store all the results).

(/me notes - I think longer term we should also cache the results found as I suspect doing all this searching for every line saved in the csv is probably costing us quite a bit of runtime...)
@MarioCarrilloA , do you have bandwidth to add the search for /usr/bin and /usr/local/bin?
If not, I'm happy if we open an Issue to note we can be smarter, and merge this so we can get the code up and running